k8s/issuer/kustomize/issuer/base/profile-mapping.json

```diff
@@ -0,0 +1 @@
+[{}]
```

Comment:
Missing newline at the end of the file

Reply:
Also, is it necessary to have this file if it's just effectively empty?

Reply:
k8s is failing without it, same pattern is used in [vcs](https://github.com/trustbloc/k8s/blob/main/vcs-v1/kustomize/vcs-v1/base/profiles.json)